tests/src/main/java/net/neoforged/neoforge/oldtest/DataGeneratorTest.java

```diff
@@ -168,8 +168,8 @@ public static void gatherData(GatherDataEvent.Client event) {
                         new WithConditions<>(new OverlayMetadataSection.OverlayEntry(new InclusiveRange<>(0, Integer.MAX_VALUE), "neoforge_overlays_test")))))
                 .add(GeneratingOverlayMetadataSection.TYPE, new GeneratingOverlayMetadataSection(List.of(
                         new WithConditions<>(new OverlayMetadataSection.OverlayEntry(new InclusiveRange<>(0, Integer.MAX_VALUE), "pack_overlays_test")),
-                        new WithConditions<>(new OverlayMetadataSection.OverlayEntry(new InclusiveRange<>(0, Integer.MAX_VALUE), "conditional_overlays_enabled"), new ModLoadedCondition("neoforge")),
-                        new WithConditions<>(new OverlayMetadataSection.OverlayEntry(new InclusiveRange<>(0, Integer.MAX_VALUE), "conditional_overlays_enabled"), new ModLoadedCondition("does_not_exist")))))
+                        new WithConditions<>(new OverlayMetadataSection.OverlayEntry(new InclusiveRange<>(0, Integer.MAX_VALUE), "conditional_overlays_enabled"), NeoForgeConditions.modLoaded(NeoForgeVersion.MOD_ID)),
```

Comment:
Note that `NeoForgeVersion` is internal and should ideally not be used outside of NeoForge. (At the moment at least).

Reply:
Sure but this is a internal test class, i see no problem using internal things here.
but if using internals is a problem, then we should move the `MOD_ID` constant to a more public location as thats the only reason im using `NeoForgeVersion` here for the mod id constant. 

I have also seen alot of people struggle to find is/think we dont have one/if they do find the constant question why its in the version class.
Clearly something people want to use and having it inside internals doesnt make sense, it should be part of the public api imo.